### ISSUE TITLE
No Focus Style on Media Library Images

### ISSUE THREAD

**Issue by jeryj** [Automattic/wp-calypso#42412]
`outline: none;` is set on all buttons, and the focus styles for the image buttons are not added back in.

#### Steps to reproduce

1. Go to the media library: https://wordpress.com/media/[subdomain].wordpress.com
2. Press `tab` until you arrive at the images
3. No focus style is present on images, so you don't know which image you're on

#### What I expected
Focus styles should be present to show where the focus is on images

#### What happened instead
There is no focus.

#### Browser / OS version
MacOS Catalina 10.15.4
Chrome 81.0.4044.138 

#### Screenshot / Video
**Showing the `activeElement` in the console is the image button**
<img width="863" alt="" src="https://user-images.githubusercontent.com/967608/82457328-e3fe4180-9a7a-11ea-83bf-f4e0394e0650.png">

**No focus style on the image button when it's the `activeElement`**
<img width="953" alt="" src="https://user-images.githubusercontent.com/967608/82457342-e791c880-9a7a-11ea-9c7c-8dcef46d723c.png">

#### Context / Source

#manual-testing

**Comment by Aurorum** [comment]
PR in #42416 for this issue! :)

**Comment by xpurichan** [comment]
This issue has been open for some time so I wanted to retest it. I can confirm that the problem is still ongoing with no focal point being added to the images.

Adding the "Type: Bug" to this report.

@Aurorum it seems the above PR may have gone stale. Is this something that we plan to complete in the future, or a "won't fix"?

**Comment by Aurorum** [comment]
Hi! I'm still happy to help work on anything, and that PR should still work, but it needs to be reviewed by someone...it's probably slipped through the cracks, but I'm not sure who to ping and ask for that review.
